packages/astro/src/assets/fonts/vite-plugin-fonts.ts

```diff
@@ -111,14 +111,22 @@ export function fontsPlugin({ settings, sync, logger }: Options): Plugin {
 
 		const root = settings.config.root;
 
+		const families = await Promise.all(
+			settings.config.experimental.fonts!.map((family: FontFamily<any>) =>
+				resolveFontFamily({
+					family,
+					root,
+					resolveMod,
+					generateNameWithHash: (_family) =>
+						`${_family.name}-${h64ToString(JSON.stringify(sortObjectByKey(_family)))}`,
+				}),
+			),
+		);
```

Comment:
Be careful here, we are swallowing possible errors coming from the resolution of fonts.  And `resolveFontFamily` throws errors!

Reply:
You mean because of `Promise.all()`?

Reply:
Yes